src/derivative_wrappers.jl

```diff
@@ -1,6 +1,7 @@
-struct SensitivityAlg{CS,AD,FDT,Jv,Quad} <: DiffEqBase.DEAlgorithm end
-Base.@pure function SensitivityAlg(;chunk_size=0,autodiff=true,diff_type=Val{:central},autojacvec=true,quad=true)
-  SensitivityAlg{chunk_size,autodiff,typeof(diff_type),autojacvec,quad}()
+struct SensitivityAlg{CS,AD,FDT,Jv,Quad,BS} <: DiffEqBase.DEAlgorithm end
+Base.@pure function SensitivityAlg(;chunk_size=0,autodiff=true,diff_type=Val{:central},autojacvec=true,quad=true,backsolve=false)
+  backsolve && (quad = false)
+  SensitivityAlg{chunk_size,autodiff,typeof(diff_type),autojacvec,quad,backsolve}()
```

Comment:
Does backsolve need to be type information?

Reply:
No, it doesn't. I will move `autojacvec,quad,backsolve` out.